src/OrchardCore.Modules/OrchardCore.Forms/Entities/Scripting/ValidateRuleMethod.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Scripting;
+
+namespace OrchardCore.Forms.Entities.Scripting
+{
+    public class ValidateRuleMethod : IGlobalMethodProvider

```

Comment:
There are a couple of small things to fix: 
![image](https://user-images.githubusercontent.com/1976647/109736063-1de0a480-7bc4-11eb-94de-2adef0249c43.png)

Also, run Visual Studio formatting (Ctrl+K+D) on the file.